src/main/java/com/akademia/projectplanner/Dto/TaskDto.java

```diff
@@ -0,0 +1,15 @@
+package com.akademia.projectplanner.Dto;
```

Comment:
skoro wszystkie nazwy pakietów zaczynamy małymi literami to tutaj też trzeba wg tej konwencji :)